core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-configuration/components/dot-experiments-configuration-goals/dot-experiments-configuration-goals.component.ts

```diff
@@ -46,8 +40,7 @@ import { DotExperimentsConfigurationGoalSelectComponent } from '../dot-experimen
         DotMessagePipeModule,
         DotDynamicDirective,
         DotIconModule,
-        UiDotIconButtonTooltipModule,
-        UiDotIconButtonModule,
+
```

Comment:
In the view of this, change from p-button-link to p-button-outlined in `Set Goal`